app/templates/explorer.hbs

```diff
@@ -13,6 +13,7 @@
     @toggleBooleanControl={{this.toggleBooleanControl}}
     @updateCompareTo={{this.updateCompareTo}}
     @isModelLoading={{this.reloadExplorerModel.isRunning}}
+    @geoType={{this.model.selection.type}}
```

Comment:
Same here, prefer argument to be `@geotype`.

Reply:
Ok, this has been updated.